### ISSUE TITLE
Tooltips are missing from the formatting option buttons

### ISSUE THREAD

**Issue by SoftVision-EmilPasca** [mozilla/notes#72]
[Affected versions]:
- Firefox 53.0 and up
- Firefox Notes v1.2

[Affected Platforms]:
- All Windows
- All Linux
- All Mac

[Prerequisites]:
- Have a Firefox profile with the latest Notes add-on version (1.2 -dev, built on 16/06/2017).

[Steps to reproduce]:
1. Open the browser with the profile from prerequisites.
2. Hover the formatting option buttons("Bold","Italic","Ordered list", etc.) from "Notes" sidebar and observe.

[Expected result]:
- The corresponding tooltip is displayed for each formatting option button.

[Actual result]:
- No tooltip is displayed.

[Notes]:
- Attached a screen recording of the issue:
![notes_tooltips_missing](https://user-images.githubusercontent.com/20083658/27226679-cef18f4e-52a8-11e7-9d41-bb100633157f.gif)

**Comment by vladikoff** [comment]
@SoftVision-EmilPasca good find, thank you!

> added this to the Stretch milestone

**Comment by Natim** [comment]
Refs https://quilljs.com/playground/#snow-toolbar-tooltips

**Comment by cedricium** [comment]
I tried adding tooltips as shown in the reference but then realized that effect is only possible using Bootstrap (see this Quill issue: https://github.com/quilljs/quill/issues/650). I went ahead and instead added titles to the buttons which shows the function name and the key-binding for that specific function.

In addition to the titles, I added three key-bindings for the 'strike', 'numbered list', and 'bulleted list' since Quill does not have default bindings for them. In my excitement to try and fix this issue I forgot to ask if I could give this a shot so hopefully it'll be alright. I've made a PR with my changes.
